pkg/agent/proxy/proxier_windows.go

```diff
@@ -44,3 +45,21 @@ func (p *proxier) uninstallLoadBalancerServiceFlows(svcIP net.IP, svcPort uint16
 	}
 	return nil
 }
+
+func (p *proxier) addClusterIPServiceRoutes(podCIDR *net.IPNet, peerNodeName string, peerNodeIP net.IP, gwConfig *config.GatewayConfig) error {
```

Comment:
I think the parameter names are not exactly right :) podCIDR for a Service ClusterIP?

Reply:
Updated parameter of `AddServiceRoutes` to `svcIP`. Also `addClusterIPServiceRoutes`.